Documentation/MigrationGuide.md

```diff
@@ -1,15 +1,116 @@
-# Master
+# SwiftGen 6.0 Migration Guide
+
+If you're migrating from SwiftGen 5.x to SwiftGen 6.0, there might be some migration steps you'll need to use.
+
+Below is a list of pointers to help you migrate to the new SwiftGen 6.0
+
+## `storyboards` command has been renamed
+
+The `storyboards` subcommand has been renamed `ib` (for Interface Builder).
+You should replace invocations in your config files with the new `ib` command name, or command line invocations of `swiftgen storyboards …` with `swiftgen ib …`.
+
+❗️ See below: the bundled `storyboards` templates have been split up into one for scenes and one for segues.
+
+## Bundled Templates have been renamed & cleaned
+
+We've (finally) removed the bundled `swift2` templates. These have been deprecated for a while and were not being tested, so we're dropping them with this release. We assume most of you are already developing in a newer version of Swift (3 or 4).
+
+The biggest change is that the `storyboards` (`ib`) templates have been split up into separate templates, grouping functionality into a specific template. There's now a template for your scene information `scenes-swift3/4`, and a template for your segue information `segues-swift3/4`. We've split these up in preparation for more upcoming functionality, such as accessibility labels. To help you generating multiple outputs, see [below](#commands-can-have-multiple-outputs).
```

Comment:
Maybe that paragraph is so important that it deserves its own title? Like splitting `## The Storyboards template has been split into two scenes + segues templates`(dedicated to that paragraph) from  `## Templates cleanup` (swift 2 removal + link to templates migration guide)